public/main.css

```diff
@@ -160,31 +174,61 @@ tbody {
 }
 
 /**   upload-progress   **/
-#progress-bar {
-  width: 300px;
-  height: 5px;
-  background: linear-gradient(
-    90deg,
-    #FD9800,
-    #D73000 var(--progress),
-    white var(--progress)
-  );
-  border: 0.5px solid;
-  border-radius: 5px;
+.progress-bar {
+  margin-top: 3px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
 }
 
-/**   share-link    **/
-.share-window {
+.percentage {
+  position: absolute;
+}
+
+.percent-number {
+  font-size: 43.2px;
+  letter-spacing: -0.78px;
```

Comment:
this and font-family can be rules on `.percentage` to affect both children, then you won't have to write them twice.